control-plane-operator/main.go

```diff
@@ -266,12 +264,16 @@ func NewStartCommand() *cobra.Command {
 			"SOCKS5_PROXY_IMAGE":                   &socks5ProxyImage,
 			"AVAILABILITY_PROBER_IMAGE":            &availabilityProberImage,
 			"TOKEN_MINTER_IMAGE":                   &tokenMinterImage,
+			"HYPERSHIFT_FEATURESET":                &featureSet,
```

Comment:
given the number of consumptions for HYPERSHIFT_FEATURESET, we might want to put that into an exported var within support/